Donation/DonationAmountView.swift

```diff
@@ -0,0 +1,200 @@
+/*****************************************************************************
+ * DonationViewController.swift
+ * VLC for iOS
+ *****************************************************************************
+ * Copyright (c) 2018 VideoLAN. All rights reserved.
+ * $Id$
+ *
+ * Authors: Carola Nitz <nitz.carola # googlemail.com>
+ *
```

Comment:
Extra line compared to other files in this PR such as `DonationButton`.
Either it's `DonationButton`'s header that needs to be adjusted or this one but let's be detail oriented and have consistency!